src/actions.ts

```diff
@@ -904,7 +904,16 @@ export const actions: {[k: string]: QueryHandler} = {
 		}
 		return {password: pw};
 	},
-
+	async 'replays/batch.json'(params) {
```

Comment:
Two things - one, can you break this change into a separate PR? 
Two - strike the .json from the name, for loginserver APIs we don't use that.